internal/plugin_test.go

```diff
@@ -372,4 +372,26 @@ func testHookFunc(t *testing.T, factory func() (*Manager, *LuaPlugin, error)) {
 			t.Fatal(err)
 		}
 	})
+
+	t.Run("LoadAllSdk", func(t *testing.T) {
```

Comment:
This test is meaningless, it will never fail.

Reply:
Sorry, I didn't understand what you meant. The prI solved was the problem of the results returned by `LoadAllSdk` being out of order,

Reply:
Because in the first iteration, you assign `sdks[0].Name` to `firstSdkName`, and in subsequent iterations, you compare `sdks[0].Name` with `firstSdkName`, they will always be equal.

Reply:
As you can see, I looped `LoadAllSdk` 10 times. The first time, I simply assigned `sdks[0]` to `firstSdkName`. Starting from the second loop, I compared `sdks[0]` with `firstSdkName` to check if they are equal. If they are not equal, it indicates that the order of the SDKs returned by `LoadAllSdk` has changed.

Reply:
Alright, that's something I didn't think of.